homeassistant/components/aseko_pool_live/config_flow.py

```diff
@@ -101,18 +101,18 @@ async def async_store_credentials(self, info: dict[str, Any]) -> ConfigFlowResul
         )
 
     async def async_step_reauth(
-        self, user_input: Mapping[str, Any]
+        self, entry_data: Mapping[str, Any]
     ) -> ConfigFlowResult:
         """Perform reauth upon an API authentication error."""
 
         self.reauth_entry = self.hass.config_entries.async_get_entry(
             self.context["entry_id"]
         )
 
-        return await self.async_step_reauth_confirm(user_input)
+        return await self.async_step_reauth_confirm()
```

Comment:
Small change in behavior...
Previously, the original credentials were tested, now the user is prompted for new credentials without attempting the old ones.
The logic with `if user_input is not None:` implies that this was the desired behavior.